simd/unit_tests/include/TestSIMD_MathOps.hpp

```diff
@@ -316,6 +316,14 @@ TEST(simd, device_math_ops) {
       << "skipping because of a non-deterministic failure reporting: "
          "Failure to synchronize stream (nil): Error in "
          "cuStreamSynchronize: an illegal memory access was encountered";
+#endif
+#ifdef KOKKOS_ENABLE_OPENACC
+#ifdef KOKKOS_COMPILER_CLANG  // FIXME_CLACC
```

Comment:
I would prefer
```suggestion
#if defined(KOKKOS_ENABLE_OPENACC) && defined(KOKKOS_COMPILER_CLANG)  // FIXME_CLACC
```
(not formatted and need to delete `#endif`)
Same comment in the next file.

Reply:
Updated TestSIMD_Reductions.hpp too.